### DISCUSSION TITLE
install/community/k8s/k8s-dce-deploy/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#3212] [Announcements]
# install/community/k8s/k8s-dce-deploy/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/install/community/k8s/k8s-dce-deploy/

**Comment by rml204** [comment]
我在使用这种方式安装的时候，报如下错误 ：
3:37PM: ready.go:277: [debug] Deployment is not ready: kpanda-system/kpanda-bindings-syncer. 0 out of 1 expected pods are ready
3:37PM: ready.go:277: [debug] Deployment is not ready: kpanda-system/kpanda-bindings-syncer. 0 out of 1 expected pods are ready
3:37PM: ready.go:277: [debug] Deployment is not ready: kpanda-system/kpanda-bindings-syncer. 0 out of 1 expected pods are ready
3:37PM: ready.go:277: [debug] Deployment is not ready: kpanda-system/kpanda-bindings-syncer. 0 out of 1 expected pods are ready
3:37PM: ready.go:277: [debug] Deployment is not ready: kpanda-system/kpanda-bindings-syncer. 0 out of 1 expected pods are ready
3:37PM: ready.go:277: [debug] Deployment is not ready: kpanda-system/kpanda-bindings-syncer. 0 out of 1 expected pods are ready
3:37PM: ready.go:277: [debug] Deployment is not ready: kpanda-system/kpanda-bindings-syncer. 0 out of 1 expected pods are ready
3:37PM: ready.go:277: [debug] Deployment is not ready: kpanda-system/kpanda-bindings-syncer. 0 out of 1 expected pods are ready
3:37PM: ready.go:277: [debug] Deployment is not ready: kpanda-system/kpanda-bindings-syncer. 0 out of 1 expected pods are ready
3:37PM: ready.go:277: [debug] Deployment is not ready: kpanda-system/kpanda-bindings-syncer. 0 out of 1 expected pods are ready
3:37PM: + exit 3
3:37PM: + cleanup
3:37PM: + echo cleanup
3:37PM: + set +ex
3:37PM: ready.go:277: [debug] Deployment is not ready: kpanda-system/kpanda-bindings-syncer. 0 out of 1 expected pods are ready
3:37PM: ready.go:277: [debug] Deployment is not ready: kpanda-system/kpanda-bindings-syncer. 0 out of 1 expected pods are ready
3:37PM: ready.go:277: [debug] Deployment is not ready: kpanda-system/kpanda-bindings-syncer. 0 out of 1 expected pods are ready
3:37PM: ready.go:277: [debug] Deployment is not ready: kpanda-system/kpanda-bindings-syncer. 0 out of 1 expected pods are ready
3:37PM: ready.go:277: [debug] Deployment is not ready: kpanda-system/kpanda-bindings-syncer. 0 out of 1 expected pods are ready
3:37PM: ready.go:277: [debug] Deployment is not ready: kpanda-system/kpanda-bindings-syncer. 0 out of 1 expected pods are ready
3:37PM: ready.go:277: [debug] Deployment is not ready: kpanda-system/kpanda-bindings-syncer. 0 out of 1 expected pods are ready
3:37PM: ready.go:277: [debug] Deployment is not ready: kpanda-system/kpanda-bindings-syncer. 0 out of 1 expected pods are ready
3:37PM: ready.go:277: [debug] Deployment is not ready: kpanda-system/kpanda-bindings-syncer. 0 out of 1 expected pods are ready
3:37PM: ready.go:277: [debug] Deployment is not ready: kpanda-system/kpanda-bindings-syncer. 0 out of 1 expected pods are ready
3:37PM: ready.go:277: [debug] Deployment is not ready: kpanda-system/kpanda-bindings-syncer. 0 out of 1 expected pods are ready
3:37PM: ready.go:277: [debug] Deployment is not ready: kpanda-system/kpanda-clusterpedia-apiserver. 0 out of 1 expected pods are ready
3:37PM: ready.go:277: [debug] Deployment is not ready: kpanda-system/kpanda-clusterpedia-apiserver. 0 out of 1 expected pods are ready
3:37PM: ready.go:277: [debug] Deployment is not ready: kpanda-system/kpanda-clusterpedia-apiserver. 0 out of 1 expected pods are ready
3:37PM: ready.go:277: [debug] Deployment is not ready: kpanda-system/kpanda-clusterpedia-apiserver. 0 out of 1 expected pods are ready
3:37PM: ready.go:277: [debug] Deployment is not ready: kpanda-system/kpanda-clusterpedia-apiserver. 0 out of 1 expected pods are ready
3:37PM: ready.go:277: [debug] Deployment is not ready: kpanda-system/kpanda-clusterpedia-apiserver. 0 out of 1 expected pods are ready
3:37PM: ready.go:277: [debug] Deployment is not ready: kpanda-system/kpanda-clusterpedia-apiserver. 0 out of 1 expected pods are ready
3:37PM: Error: timed out waiting for the condition
3:37PM: helm.go:84: [debug] timed out waiting for the condition
3:37PM: cleanup
3:37PM: [sched-info] Sched shutdown...
3:37PM: --------------------------------------
3:37PM: Unfortunately, the dce5 installation failed at step [12] ....
3:37PM:
3:37PM: You can add -j12+ to your installation command to retry from last step.
3:37PM: --------------------------------------
[Error]:Failed to Install DCE5, check above error messages. shell execution err is exit status 3

请问这是什么问题？如何解决？

**Reply by samzong** [reply]
看下来是：  kpanda-system/kpanda-bindings-syncer 这个 deployment 一直没有 ready；您可以新开一个终端，使用 `kubectl describe` 和 `kubectl logs` 查看下这个 Pod 的更多信息，进一步分析。

如果您有更多问题或者仍旧未解决，建议通过`下载中心` 页面的二维码加入我们的用户群，将会有专门的支持同学，为您全程解决问题。

**Comment by nora-pxh** [comment]
请问提示timeout了，该怎么解决呢？
10:54AM: [sched-info] Sched startup...
10:54AM: [sched-info] tasks holder at /var/lib/dce5/tmp/630302.F4L3jxPBfV
10:54AM: [Info] The precheck has been run before, maybe the installation is re-entered again. So skip checks this time.
10:54AM: Skip push images since official-service mode
10:54AM: 🔔 Install basic infrastructure applications on cluster.. 🚛
10:54AM: === Install Istio Infrastructure 1.16.1
10:54AM:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
10:54AM: 100   151  100   151    0     0   1724      0 --:--:-- --:--:-- --:--:--  1735
10:54AM: 100 24.4M  100 24.4M    0     0  3612k      0  0:00:06  0:00:06 --:--:-- 3740k
10:54AM:  Installing Istio-base..
10:54AM: tar: istio-1.16.1/manifests/charts/base: Cannot read: Is a directory
10:54AM: tar: At beginning of tape, quitting now
10:54AM: tar: Error is not recoverable: exiting now
10:54AM: Release "istio-base" has been upgraded. Happy Helming!
10:54AM: NAME: istio-base
10:54AM: LAST DEPLOYED: Thu Mar 21 10:54:17 2024
10:54AM: NAMESPACE: istio-system
10:54AM: STATUS: deployed
10:54AM: REVISION: 4
10:54AM: TEST SUITE: None
10:54AM: NOTES:
10:54AM: Istio base successfully installed!
10:54AM:
10:54AM: To learn more about the release, try:
10:54AM:   $ helm status istio-base
10:54AM:   $ helm get all istio-base
10:54AM:  Installing Istiod ..
10:54AM: tar: istio-1.16.1/manifests/charts/istio-control/istio-discovery: Cannot read: Is a directory
10:54AM: tar: At beginning of tape, quitting now
10:54AM: tar: Error is not recoverable: exiting now
10:59AM: cleanup
10:59AM: Error: UPGRADE FAILED: timed out waiting for the condition
10:59AM: [sched-info] Sched shutdown...
10:59AM: --------------------------------------
10:59AM: Unfortunately, the dce5 installation failed at step [8] ....
10:59AM:
10:59AM: You can add -j8+ to your installation command to retry from last step.
10:59AM: --------------------------------------
[Error]:Failed to Install DCE5, check above error messages. shell execution err is exit status 1

**Comment by jackmayun** [comment]
DCE安装完成后，进入到首页(即本文档最后一张截图)后，无数据显示。但是在【可观测性】模块可以看到数据，请问一般是哪里的问题？

**Comment by starrysky-hub** [comment]
7:58AM:     [Error] : There is no storageclass in th Kubernetes
7:58AM:               Please ensure default storageclass has been set up!
7:58AM: ❌ [Fail] case storageclass
7:58AM: --------- PreCheck completed--------------
7:58AM: [Error]🔥 At least one  pre-check  case failed,  Please address them before installation!
[Error]:Failed to Install DCE5, check above error messages. shell execution err is exit status 1

wxyg@master:~/DaoCloud$ kubectl get storageclass
NAME                   PROVISIONER             RECLAIMPOLICY   VOLUMEBINDINGMODE      ALLOWVOLUMEEXPANSION   AGE
local-path (default)   rancher.io/local-path   Delete          WaitForFirstConsumer   true                   41m
已经配置了默认存储，为什么安装DCE5时还是报没有默认存储？
